### ISSUE TITLE
--output-move fails

### ISSUE THREAD

**Issue by pjz** [AlBundy33/hb_rb#1]
Is `--output-move` not fully supported?

```
$ hb.rb --input /dev/sr0 --output somefile.mkv --movie --output-move /some/dest/dir
.
.
.

/home/pj/bin/software/hb_rb/lib/commands.rb:75:in `run': undefined method `move' for File:Class (NoMethodError)
    from /home/pj/bin/software/hb_rb/lib/hb_lib.rb:1265:in `block (2 levels) in convert'
    from /home/pj/bin/software/hb_rb/lib/hb_lib.rb:1261:in `each'
    from /home/pj/bin/software/hb_rb/lib/hb_lib.rb:1261:in `block in convert'
    from /home/pj/bin/software/hb_rb/lib/hb_lib.rb:859:in `each'
    from /home/pj/bin/software/hb_rb/lib/hb_lib.rb:859:in `convert'
    from ./hb.rb:66:in `block (2 levels) in <main>'
    from ./hb.rb:58:in `each'
    from ./hb.rb:58:in `block in <main>'
    from ./hb.rb:46:in `each'
    from ./hb.rb:46:in `<main>'
```

**Comment by AlBundy33** [comment]
Sorry for late response - it seems that File::move was only available up to ruby 1.8.

I'll check this

**Comment by AlBundy33** [comment]
I'm not sure if I have done anything. 
I close this issue - if there are still problems simply create a new issue. ;-)
